api/src/main/java/jakarta/json/bind/annotation/JsonbProperty.java

```diff
@@ -51,6 +51,8 @@
      * Switches o/off serialization of null values.
      *
      * @return True if field with null value should be serialized as key/value pair into JSON with null value.
+     * @deprecated Please use {@link JsonbNillable} as another annotation instead. This option will be removed in the future.
      */
+    @Deprecated(since = "2.1.0")
```

Comment:
Specs do not have service versions, only API jars. I guess here should be since="2.1".

Reply:
Aaaah that's right. Thank you.